sdk/eventhub/azure-eventhubs/tests/livetest/asynctests/test_negative_async.py

```diff
@@ -16,228 +16,129 @@
     AuthenticationError,
     EventDataSendError,
 )
-from azure.eventhub.aio.client_async import EventHubClient
+from azure.eventhub.aio import EventHubConsumerClient, EventHubProducerClient
 
 
 @pytest.mark.liveTest
 @pytest.mark.asyncio
 async def test_send_with_invalid_hostname_async(invalid_hostname, connstr_receivers):
     _, receivers = connstr_receivers
-    client = EventHubClient.from_connection_string(invalid_hostname)
-    sender = client._create_producer()
-    with pytest.raises(AuthenticationError):
-        await sender.send(EventData("test data"))
-    await sender.close()
-    await client.close()
+    client = EventHubProducerClient.from_connection_string(invalid_hostname)
+    async with client:
+        with pytest.raises(ConnectError):
+            await client.send(EventData("test data"))
 
 
+@pytest.mark.parametrize("invalid_place",
+                         ["hostname", "key_name", "access_key", "event_hub", "partition"])
 @pytest.mark.liveTest
 @pytest.mark.asyncio
-async def test_receive_with_invalid_hostname_async(invalid_hostname):
-    client = EventHubClient.from_connection_string(invalid_hostname)
-    receiver = client._create_consumer(consumer_group="$default", partition_id="0", event_position=EventPosition("-1"))
-    with pytest.raises(AuthenticationError):
-        await receiver.receive(timeout=3)
-    await receiver.close()
-    await client.close()
-
+async def test_receive_with_invalid_param_async(live_eventhub_config, invalid_place):
+    eventhub_config = live_eventhub_config.copy()
+    if invalid_place != "partition":
+        eventhub_config[invalid_place] = "invalid " + invalid_place
+    conn_str = "Endpoint=sb://{}/;SharedAccessKeyName={};SharedAccessKey={};EntityPath={}".format(
```

Comment:
This complicated format string shows up as a literal in a few different places. The format sting could be a "constant", or there could be a helper function for generating these connection strings